docs/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,175 @@
+:pyenv: link:https://github.com/OpenZeppelin/cairo-contracts/blob/release-v0.4.0b/src/openzeppelin/access/ownable/library.cairo[pyenv]
+
+= Overview
+
+Nile is a CLI tool to develop or interact with StarkNet projects written in Cairo. It consists of different components for developing, compiling, testing, and deploying your smart contracts and dApps, providing a CLI for executing tasks, and a Runtime Environment (NRE) for scripting.
+
+The package is designed to be extensible and very customizable by using plugins. This guide will take you through the installation of our recommended setup, but as we expect a lot of functionality coming from plugins, you are free to customize it.
+
+TIP: You can get a list of plugins extending Nile functionality from the xref:plugins.adoc[Plugins] section.
+
+:oz-account: link:https://github.com/OpenZeppelin/cairo-contracts/blob/main/src/openzeppelin/account/presets/Account.cairo[Openzeppelin Account]
+
+== Quickstart
+
+Using a sample contract, we will explore the basics of creating a Nile project. We will test it locally, deploy an account and the contract to a devnet node, and send transactions through the account to the contract.
```

Comment:
piggybacking on [Andrew's comments](https://github.com/OpenZeppelin/nile/pull/298#discussion_r1031075746), here we mention the node but we never show how to use it.

i'll mix that with the following suggestion:

<img width="208" alt="image" src="https://user-images.githubusercontent.com/1703831/209575694-9d5af8d4-0400-484e-8cb3-fe2913d2729e.png">


I think everything below `Compiling` should be a subsection of a wider "Interacting with a node" or "with StarkNet" section. and i would add an additional subsection for scripting, something like "Custom scripts" or "Custom tasks"

Reply:
I added an "Interacting with Starknet" section including running a local node, the only caveat I find is that we lose the index entries (in the section to the right).

For the custom scripts not sure what to add, when Quickstart's next page is NRE and scripting already.